server/tests/test_helper.py

```diff
@@ -0,0 +1,33 @@
+import tabby_server.services.helper as helper
+from tests.variable import test_base64
+import os
+import unittest
+
+
+class Helper (unittest.TestCase):
```

Comment:
We use `pytest`, so test functions do not need to be in a class. Pytest runs all functions which start with `test_`.